javaparser-core/src/main/javacc/java_1_8.jj

```diff
@@ -209,7 +209,7 @@ final class ASTParser {
     }
 
     private static class RangedList<T extends Node> {
-        Range range = Range.UNKNOWN;
+        Range range = Range.range(0, 0, 0, 0);
```

Comment:
Hmmm, that's a temporary position that should never get outside the parser... But maybe UNKNOWN is still of some use to people so I'll restore it.